CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required(VERSION 3.10.2)
+
+# where to find our CMake modules
+set(CMAKE_MODULE_PATH "$ENV{CARMA_OPT_DIR}/cmake")
```

Comment:
We should not be overwriting `CMAKE_MODULE_PATH`. We should append to it instead.

```cmake
list(APPEND CMAKE_MODULE_PATH $ENV{CAMRA_OPT_DIR}/cmake)
```
We should also check to see if its defined first:

```cmake
if(DEFINED ENV{CARMA_OPT_DIR})
  list(APPEND CMAKE_MODULE_PATH $ENV{CAMRA_OPT_DIR}/cmake)
endif()
```

Reply:
Good catch